test/jdk/javax/swing/regtesthelpers/Util.java

```diff
@@ -282,8 +319,8 @@ public static ArrayList<Integer> getSystemMnemonicKeyCodes() {
     */
     public static JDialog createModalDialogWithPassFailButtons(final String failString) {
         JDialog  retDialog = new JDialog();
-        Box      buttonBox = Box.createHorizontalBox();
-        JButton  passButton = new JButton("Pass");
+        Box buttonBox = Box.createHorizontalBox();
+        JButton passButton = new JButton("Pass");
         JButton  failButton = new JButton("Fail");
```

Comment:
I'd leave these lines untouched.

Usages of `createModalDialogWithPassFailButtons` could be replaced with `PassFailJFrame` for consistency. It's out of scope.